src/assets/styles/includes.scss

```diff
@@ -23,3 +23,21 @@
   word-wrap: break-word;
   word-break: break-word;
 }
+
+// mixin to hide inputs from form-associated components
+@mixin hidden-form-input() {
+  ::slotted(input[slot="hidden-form-input"]) {
+    bottom: 0 !important;
+    left: 0 !important;
+    margin: 0 !important;
+    opacity: 0 !important;
+    outline: none !important;
+    padding: 0 !important;
+    position: absolute !important;
+    right: 0 !important;
+    top: 0 !important;
+    transform: none !important;
+    -webkit-appearance: none !important;
+    z-index: -1 !important;
+  }
```

Comment:
What is our plan for this when we eventually deprecate Sass?

Reply:
Unclear right now, but I'm guessing we could use [PostCSS mixins](https://github.com/postcss/postcss-mixins) for this.